kectl/app/cmd/util/cloudinstaller.go

```diff
@@ -0,0 +1,261 @@
+package util
+
+import (
+	"bufio"
+	"bytes"
+	"fmt"
+	"io"
+	"io/ioutil"
+	"os"
+	"os/exec"
+	"time"
+)
+
+type KubeCloudInstTool struct {
+	Common
+}
+
+func (cu *KubeCloudInstTool) InstallTools() error {
+	cu.SetOSInterface(GetOSInterface())
+	cu.SetKubeEdgeVersion(cu.ToolVersion)
+
+	err := cu.InstallKubeEdge()
+	if err != nil {
+
+		return err
+	}
+	fmt.Println("Installation of kubeedge package is sucessfull")
+	err = cu.generatecertificates()
+	if err != nil {
+		fmt.Println(" in err")
+		return err
+	}
+	fmt.Println("Certificates got genertaed and its kept in /etc/kubeedge/kubeedge certificates folder")
```

Comment:
genertaed -> generated

Reply:
This log could be shortened : Certificates got generated at : /etc/kubeedge/kubeedge